openquakeplatform/openquakeplatform/icebox/templates/icebox.html

```diff
@@ -97,7 +99,7 @@
             <% } %>
           <% } %>
 
-          <% if(calc.get('status') == 'complete' || calc.get('status') == 'failed' || calc.get('status') == 'queued') { %>
+          <% if(calc.get('status')) { %>
```

Comment:
So, if you have a status, whatever it is, you want the final href with the calc id to be there, otherwise you want the button to display the traceback in case the status it `failed`. If you don't have a status, you don't want to display the calc id. Am I understanding correctly?